www-client/dooble/dooble-2023.07.15.recipe

```diff
@@ -77,8 +78,8 @@ INSTALL()
 	cp -f Dooble $appsDir/Dooble
 
 	local MAJOR="2023"
-	local MIDDLE="02"
-	local MINOR="25"
+	local MIDDLE="04"
+	local MINOR="20"
```

Comment:
This doesn't match the recipe version.

Suggestion: other recipes use `echo $portVersion | cut -d. -f{1,2,3}` for this (for `{MAJOR,MIDDLE,MINOR} respectively), so that no change is needed on update.

Reply:
Fixed